core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -225,6 +242,20 @@ public void resetContext() throws Exception {
     verify(mMockFileSystemContext).reset(alluxio.Configuration.global());
   }
 
+  /**
+   * Tests that initializing the {@link AbstractFileSystem} with Alluxio on Zookeeper URI
```

Comment:
If possible try to capture the description only in the method name so that javadoc isn't necessary

Reply:
Removed, name is clear enough.